examples/clipping_planes.py

```diff
@@ -39,7 +39,7 @@ def create_scene(clipping_planes, clipping_mode):
 camera.position.z = 250
 
 controls = gfx.OrbitControls(camera.position.clone())
-controls.add_default_event_handlers(canvas, camera)
+controls.add_default_event_handlers(renderer, canvas, camera)
```

Comment:
I wonder whether we can remove the canvas from this list, since the renderer already has a reference to it. Handlers need it for the size and for requesting a new draw. Maybe we can replicate these methods on the renderer? We could leave this for after this PR - its big enough as it is.